interop-testing/src/main/java/io/grpc/testing/integration/TestLogHandler.java

```diff
@@ -0,0 +1,37 @@
+package io.grpc.testing.integration;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.logging.Handler;
+import java.util.logging.LogRecord;
+
+public class TestLogHandler extends Handler {
+
+  private ArrayList<Integer> windows = new ArrayList<Integer>();
+
+  @Override
+  public void publish(LogRecord record) {
+    String message = record.getMessage();
+    if (message.contains("Window")) {
+      windows.add(Integer.parseInt(record.getParameters()[0].toString()));
+    }
+  }
+
+  @Override
+  public void flush() {
+    // TODO(mcripps): Auto-generated method stub
```

Comment:
Haven't implemented anything other than publish yet because that seems to be all I need. Is it necessary to implement the other functions?